test/csuite/wt8246_compact_rts_data_correctness/main.c

```diff
@@ -337,12 +339,14 @@ large_updates(WT_SESSION *session, const char *uri, char *value, int commit_ts)
         cursor->set_key(cursor, i + 1);
         val = (uint64_t)__wt_random(&rnd);
         cursor->set_value(cursor, val, val, val, value);
-        if ((ret = cursor->insert(cursor)) == WT_ROLLBACK)
```

Comment:
I don't think this is correct. We need to rollback the transaction first and then retry.

Reply:
Fixed 457b8b94a160b3e70f626937ccbd32dd1c47e471